src/components/intro.tsx

```diff
@@ -6,7 +6,7 @@ const Intro = () => {
       <h1 className="text-6xl md:text-8xl font-bold tracking-tighter leading-tight md:pr-8">
         {BLOG_TITLE}.
       </h1>
-      <h4 className="text-center md:text-left text-lg mt-5 md:pl-8">
+      <h4 className="text-center text-secondary md:text-left text-lg mt-5 md:pl-8">
```

Comment:
This text lack of contrast:

![image](https://user-images.githubusercontent.com/95120/109936840-9a5faa00-7cce-11eb-82f9-c72120559f8e.png)